tests/components/verisure/test_config_flow.py

```diff
@@ -138,9 +147,19 @@ async def test_full_user_flow_single_installation_with_mfa(
     assert "flow_id" in result2
 
     mock_verisure_config_flow.login.side_effect = None
-    mock_verisure_config_flow.installations = [
-        mock_verisure_config_flow.installations[0]
-    ]
+
+    def func():
+        return {
+            "data": {
+                "account": {
+                    "installations": [
+                        {"giid": "12345", "alias": "ascending"},
+                    ]
+                }
+            }
+        }
+
+    mock_verisure_config_flow.get_installations = func
```

Comment:
Get the first item from the mocked return value and assign that instead (no need to define things twice).

Reply:
@frenck Can you hlep? How can I go from this
```py
{
    "data": {
        "account": {
            "installations": [
                {
                    "giid": "12345",
                    "alias": "ascending",
                    "address": {"street": "12345th street"},
                },
                {
                    "giid": "54321",
                    "alias": "descending",
                    "address": {"street": "54321th street"},
                },
            ]
        }
    }
}
```

to this
```py
{
    "data": {
        "account": {
            "installations": [
                {
                    "giid": "12345",
                    "alias": "ascending",
                    "address": {"street": "12345th street"},
                },
            ]
        }
    }
}
```
without recreating it?